client/src/components/SponsorsComps/SponserGeneralComps.jsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable prettier/prettier */
+import React from "react";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPencilAlt, faTrash } from "@fortawesome/free-solid-svg-icons";
+import styled from 'styled-components';
+import { ButtonStyledLink } from '../GlobalStyledComponents/ReversimStyledComps'
+import {image} from '../../images';
+
+library.add(faPencilAlt, faTrash);
+
+//styled-components section
+
+const SponsorItem = styled.div`
+${( { theme: { space } } ) =>`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: ${space.xl} ${space.s};
+`}
+`;
+
+const ImageContainer= styled.a`
+  ${( { theme: { color, space } } ) => `
+      width: 248px;
+      height: 230px;
+      max-width: 248px;
```

Comment:
not sure we need both `max-width` and `width`

but maybe with images, there is some trick?